packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -101,16 +95,30 @@ export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget
 
         this.term = new Xterm.Terminal({
             experimentalCharAtlas: 'dynamic',
+            experimentalBufferLineImpl: 'TypedArray',
             cursorBlink: false,
-            fontFamily: cssProps.fontFamily,
-            fontSize: cssProps.fontSize,
+            fontFamily: this.preferences['terminal.integrated.fontFamily'],
+            fontSize: this.preferences['terminal.integrated.fontSize'],
+            fontWeight: this.preferences['terminal.integrated.fontWeight'],
+            fontWeightBold: this.preferences['terminal.integrated.fontWeightBold'],
+            letterSpacing: this.preferences['terminal.integrated.letterSpacing'],
+            lineHeight: this.preferences['terminal.integrated.lineHeight'],
             theme: {
                 foreground: cssProps.foreground,
                 background: cssProps.background,
                 cursor: cssProps.foreground,
                 selection: cssProps.selection
             },
         });
+        this.toDispose.push(this.preferences.onPreferenceChanged(change => {
```

Comment:
I wonder, would you expect to see changes of `terminal.integrated.fontFamily` reflected in terminals as well?

Reply:
Yes?

Reply:
Do you mean `editor.fontFamily`? No I do not expect that.

Reply:
yes, that was a typo.
ok